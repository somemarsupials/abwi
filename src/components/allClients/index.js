export { default as ClientsPage } from './clientsList';
export { default as ClientsList } from './clientsPage';
