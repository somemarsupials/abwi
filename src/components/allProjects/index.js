export { default as List } from './list';
export { default as Page } from './page';
