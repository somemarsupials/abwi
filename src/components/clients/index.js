export { default as ClientPage } from './clientPage';
export { default as ClientDetail } from './clientDetail';
