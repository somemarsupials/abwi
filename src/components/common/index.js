export { default as TitleBar } from './titleBar';
export { default as ErrorBar } from './errorBar';
