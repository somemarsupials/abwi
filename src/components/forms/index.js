export { default as ProjectForm } from './projectForm';
