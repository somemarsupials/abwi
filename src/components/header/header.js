import React from 'react';
import SiteLinks from './siteLinks';

export default function(props) {
  return (
    <nav className="navbar navbar-default">
      <ul className="nav navbar-nav">
        <a className="navbar-brand" href="/">ABWI</a>
        <SiteLinks />
      </ul>
    </nav>
  );
};
