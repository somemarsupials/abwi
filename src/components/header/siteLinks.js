import React from 'react';
import { Link } from 'react-router-dom';

export default function(props) {
  return (
    <ul className="nav navbar-nav">
      <li><Link to="/projects">Projects</Link></li>
    </ul>
  );
};
