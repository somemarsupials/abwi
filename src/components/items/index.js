export { default as ItemList } from './itemList';
