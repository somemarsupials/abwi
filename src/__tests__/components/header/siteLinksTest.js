import React from 'react';
import { shallow } from 'enzyme';

import { SiteLinks } from '../../../components/header';

describe('SiteLinks', () => {
  let siteLinks;

  describe('when passed detailed information', () => {
    beforeEach(() => {
      siteLinks = shallow(<SiteLinks />);
    });

    it('correctly renders', () => {
      expect(siteLinks).toMatchSnapshot();
    });
  });
});

