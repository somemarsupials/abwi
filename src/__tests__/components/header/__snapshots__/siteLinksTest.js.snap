// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteLinks when passed detailed information correctly renders 1`] = `
<ul
  className="nav navbar-nav"
>
  <li>
    <Link
      replace={false}
      to="/projects"
    >
      Projects
    </Link>
  </li>
</ul>
`;
